src/app/stores/migrator.ts

```diff
@@ -607,8 +607,12 @@ export class Migrator {
             if (element.properties.xData === undefined) {
               element.properties.xData = null;
             }
-            element.properties.xData.offset = 0;
-            element.properties.xData.tickFormatType = TickFormatType.None;
+            if (element.properties.xData.offset === undefined) {
+              element.properties.xData.offset = 0;
+            }
+            if (element.properties.xData.tickFormatType == undefined) {
```

Comment:
is this intentionally == or should it be === also?

Reply:
We use 'null' as value for properties. When the property of object has null value in `JSON.serialization` property name saves with `null` value, when with `undefined` property removes from JSON document.